src/util/string.h

```diff
@@ -245,6 +245,23 @@ inline std::string lowercase(const std::string &str)
 
 /**
  * @param str
+ * @return A copy of \p str converted to all uppercase characters.
+ */
+inline std::string uppercase(const std::string &str)
+{
+	std::string s2;
+
+	s2.reserve(str.size());
+
+	for (size_t i = 0; i < str.size(); i++)
+		s2 += toupper(str[i]);
+
+	return s2;
+}
```

Comment:
I  used it for stylistic reasons, as SQL keywords are normally written in uppercase.

I'll change it to lowercase.